TACC/Core-Portal
# task/FP-1549: Update node to LTS (v16.x)

## Overview: ##
https://nodejs.org/en/about/releases/

LTS is now 16.x

- Minor updates to formatting in package-lock.json from bump in node version
- Minor version bumps

## Related Jira tickets: ##

* [FP-1549](https://jira.tacc.utexas.edu/browse/FP-1549)

## Testing Steps: ##
1. run `make build`
2. run `npm ci` then `npm run dev`, and ensure everything works normally
